### PR TITLE
chore: bump version to beta.9 w/ changelog

### PR DESCRIPTION
**Pull request by jelbourn**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,308 @@
+<a name="2.0.0-beta.9"></a>
+# [2.0.0-beta.9 découpage-panjandrum](https://github.com/angular/material2/compare/2.0.0-beta.8...2.0.0-beta.9) (2017-08-29)
+
+### Highlights
+* Over 140 bug fixes
+* Nested menus
+* Autocomplete supports `md-optgroup`
+* `Overlay` moved to `@angular/cdk`
+* New `MdSelectionList`
```
Comment (kara) [inline CHANGELOG.md]: Nit: New component `MdSelectionList` ?

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,308 @@
...
+* Almost all components now use `OnPush` change detection (dialog being the exception).
+* You can now get back the `EmbeddedViewRef` when attaching a `TemplatePortal`
+* `MdSidenav` has been split into `MdSidenav` and `MdDrawer`. The `MdSidenav` is now meant to be
+used for top-level application navigation, while the drawer is meant to be used for more local
+split views. While there are no differences introduced between the two in this release, future
+releases will see different features added to each.
+
+### Breaking changes
+* Imports from `@angular/cdk` are now scoped to a specific sub-package. For example, if you
+previous had:
```
Comment (kara) [inline CHANGELOG.md]: Typo: previous -> previously

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,3 +1,308 @@
...
+```ts
+import {LiveAnnouncer, Overlay, Directionality} from '@angular/cdk';
+```
+You would will now need to write:
```
Comment (kara) [inline CHANGELOG.md]: Typo: would will now -> will now

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,3 +1,308 @@
...
+```ts
+import {LiveAnnouncer} from '@angular/cdk/a11y';
+import {Directionality} from '@angular/cdk/bidi';
+import {Overlay} from '@angular/cdk/overlay';
+```
+
+This helps ensure you're only pulling in the pieces of the cdk being used as well as providing more
+context about what an imported symbol is be used for.
```
Comment (kara) [inline CHANGELOG.md]: Typo: is be used for -> is being used for

File: CHANGELOG.md [hunk 5]
```diff
@@ -1,3 +1,308 @@
...
+empty `Map`.
+* **observe-content:** `ObserveContentModule` has been renamed to `ObserversModule`
+* **overlay:** `PositionStrategy` now has an `attach` method and no longer passes the overlay
+DOM element to `apply`.
+* **datepicker:** You must now use a date object (of whatever type your DateAdapter uses) rather
+than a string when setting the value of the datepicker programmatically (through value, ngModel,
+or formControl).
+* **datepicker:** `NativeDateAdapter` will now use Angular's LOCALE_ID instead of the browser's
+locale.
+* **sidenav:** CSS classes have changed from `mat-sidenav-` to `mat-drawer-`
```
Comment (kara) [inline CHANGELOG.md]: Nit: CSS class prefixes ?

File: CHANGELOG.md [hunk 6]
```diff
@@ -1,3 +1,308 @@
...
+|------------------|--------------|
+| md-line          | mdLine       |
+| md-tab-link      | mdTabLink    |
+| md-tab-label     | mdTabLabel   |
+| md-card-avatar   | mdCardAvatar |
+
+* `md-input-container` has been renamed to `md-form-field`. The old selector will be removed in a
+subsequent release. This is in preparation to making `md-select` a child of `md-form-field`
+such that both select and input share the same features and appearance.
+* For camcelCased directives, each corresponding `@Input()` is now also prefixed. For example,
```
Comment (kara) [inline CHANGELOG.md]: Typo: camcelCased -> camelCased

File: CHANGELOG.md (hunk 5 above)
Comment (jelbourn) [reply CHANGELOG.md]: I consider the "prefix" to just be "md"

**Comment by jelbourn** [comment]
@kara fixed typos

File: CHANGELOG.md (hunk 5 above)
Comment (BenRacicot) [reply CHANGELOG.md]: Don't forget about `mdDatepickerToggle` (looking for the thread now)

**Pull request merged**